event.go

```diff
@@ -306,7 +306,7 @@ example:
 					if strings.HasPrefix(err.Error(), "msg: auth-required:") && kr != nil && doAuth {
 						// if the relay is requesting auth and we can auth, let's do it
 						pk, _ := kr.GetPublicKey(ctx)
-						log("performing auth as %s... ", pk)
+						log("auth to publish event %s...", pk)
```

Comment:
is this fixing the issue of the auth message garbling your terminal? I don't see how